tests/capture/input_generators/test_attack_point_iterator.py

```diff
@@ -145,3 +146,113 @@ def test_attack_point_iterator_unrestricted_generator_all_args_len():
     output = list(iter(build_attack_points_iterator(config)))
     assert len(output) == config["bunches"] * config["elements"]
     assert len(output) == len(build_attack_points_iterator(config))
+
+
+def test_attack_point_iterator_repeat():
+    config = {
+        "operation": "repeat",
+        "repetitions": 2,
+        "configuration": {
+            "operation": "constants",
+            "name": "key",
+            "values": [1, 2, 3]
```

Comment:
Some tests are missing. I would like to see tests for `repetitions in [-1, 0, 1, 2]` and for `len(values) in [0, 1, 2, 3]`. Especially the edge case of `repetitions = -1` and `values = []` is not covered (documented and the code does not produce the expected result, compare with `itertools.cycle([])`).